app/views/_pod-template.html

```diff
@@ -7,194 +7,67 @@
   addHealthCheckUrl (optional)
  -->
 
- <!-- Prompt to add health checks if missing. -->
-<div ng-if="detailed && addHealthCheckUrl && !(podTemplate | hasHealthChecks)" class="alert alert-info">
-  <span class="pficon pficon-info" aria-hidden="true"></span>
-  <span ng-if="podTemplate.spec.containers.length === 1">This container has no health checks</span>
-  <span ng-if="podTemplate.spec.containers.length > 1">Not all containers have health checks</span>
-  to ensure your application is running correctly.
-  <a ng-href="{{addHealthCheckUrl}}" class="nowrap">Add Health Checks</a>
-</div>
+<h3>Template</h3>
```

Comment:
We can't pull the header into this directive since it will show up on the project overview where this directive is also used.